polus-apply-flatfield-plugin/run-plugin.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+version=$(<VERSION)
+datapath=$(readlink --canonicalize ../data)
+echo ${datapath}
+
+# Inputs
+imgDir=/data/input
+imgPattern=r01c01f{p+}p0{z}-ch1sk1fk1fl1.ome.tif
+ffDir=/data/output_basic
+brightPattern=r01c01f\(001-121\)p0{z}-ch1sk1fk1fl1_flatfield.ome.tif
+darkPattern=r01c01f\(001-121\)p0{z}-ch1sk1fk1fl1_darkfield.ome.tif
```

Comment:
Instead of using `\(` notation to enable using input values with brackets, I think it would be easier to put the value inside of quotes `''` in the docker run command.   
 For example : `docker run ......--pattern x(001-003)_y(001-003).ome.tif` will throw an error but `docker run ......--pattern 'x(001-003)_y(001-003).ome.tif'` will not.